cfme/tests/middleware/test_middleware_provider.py

```diff
@@ -43,7 +44,7 @@ def test_add_cancelled_validation():
 def test_password_mismatch_validation(soft_assert):
     """ Tests password mismatch check """
     prov = HawkularProvider()
-    cred = prov.Credential(
+    cred = Credential(
```

Comment:
actually, Credential class was moved outside of provider class. So, this change is correct. But you have pointed out to other statements which were missed for some reason. I've fixed these statements.